### ISSUE TITLE
username unknown when logging in via oauth 

### ISSUE THREAD

**Issue by hdeadman** [bonny/WordPress-Simple-History#40]
I just started using an oauth plugin that allowed me to login to wordpress using my google account. The message in simple-history was "Unknown user logged in warning". The details contained my actual user login. The oauth plugin i used is perrybutler/WP-OAuth in github.com. The _user_login field in the message details is correct. 

```
id 	9613
logger 	SimpleUserLogger
level 	warning
date 	2015-01-17 21:17:17
message 	Unknown user logged in
type 	
initiator 	wp_user
occasionsID 	706f252c2c3e693a80cb250d010a398e
subsequentOccasions 	1
rep 	1
repeated 	1
occasionsIDType 	706f252c2c3e693a80cb250d010a398e
context_message_key 	user_unknown_logged_in
_message_key 	user_unknown_logged_in
_user_id 	1
_user_login 	hdeadman
_user_email 	abc.lastname@xxx.com
_server_remote_addr 	123.123.123.123
```

**Comment by bonny** [comment]
Thanks for the bug report. I tried the plugin quickly but where unable to get it to work. Will try again later..

**Comment by hdeadman** [comment]
If you want to try get the plugin working to test it out, here is some info from when I set it up to work with a Google login. 

Visit the Google website for developers https://console.developers.google.com and create a new project. 

Enable Google+ API
![image](https://cloud.githubusercontent.com/assets/1556777/5808773/1fef80b8-9fff-11e4-8e75-6c0fa827ea0c.png)

Create new Oauth client under credentials - (any redirect url should end in slash)
![image](https://cloud.githubusercontent.com/assets/1556777/5808733/be862d36-9ffe-11e4-9553-0b76f60af8c0.png)

Configure consent screen with e-mail and site url 
![image](https://cloud.githubusercontent.com/assets/1556777/5808817/a9a5c7c2-9fff-11e4-9cbe-75aef4d8094e.png)

Enter the client id and client secret in wordpress - 
![image](https://cloud.githubusercontent.com/assets/1556777/5808760/fc7fc1c4-9ffe-11e4-92af-254d195633a1.png)

Link a wordpress account with a Google account (in wordpress users at bottom of profile)
![image](https://cloud.githubusercontent.com/assets/1556777/5808831/d7df6b3e-9fff-11e4-9afd-1b1aed6f376d.png)

I initially got a 500 error when I tried to login b/c my php site didn't have php5-curl package installed. The plugin has an option to do http requests a different way but I just installed the package.

**Comment by bonny** [comment]
Thanks for the guide! I had missed the last part with "Linked Accounts". Now I can log in with Google and I see the same thing as you: "unknown user logged in".

**Comment by bonny** [comment]
I added a issue to the WP-OAuth plugin (https://github.com/perrybutler/WP-OAuth/issues/12). Maybe he will fix it on his side, but I will take another look at it on my side too.

**Comment by perrybutler** [comment]
This was fixed in the latest WP-OAuth update (v0.3.1).

Before and after the fix:

![simple-history](https://cloud.githubusercontent.com/assets/3649315/6136279/3877538c-b124-11e4-89a0-6c6144bf9ae5.JPG)

**Comment by bonny** [comment]
Thanks × 2! :)
